validator/src/main/java/com/amazon/aoc/services/XRayService.java

```diff
@@ -26,7 +26,9 @@
 import java.util.Date;
 import java.util.List;
 import org.joda.time.DateTime;
+import lombok.extern.log4j.Log4j2;
 
+@Log4j2
```

Comment:
Why do we need `Log4j2` annotation?

Reply:
It was used for debugging purpose. Removed.